descriptors/ModuleDescriptor-template.json

```diff
@@ -521,7 +522,8 @@
       { "name": "SCORE_PART_NUMBER_LAST", "value": "0" },
       { "name": "SCORE_PART_NUMBER_LAST_REFERENCE", "value": "100" },
       { "name": "SYSTEM_PROCESSING_USERNAME", "value": "data-import-system-user" },
-      { "name": "SYSTEM_PROCESSING_PASSWORD", "value": "data-import-system-user" }
+      { "name": "SYSTEM_PROCESSING_PASSWORD", "value": "data-import-system-user" },
+      { "name": "ASYNC_PROCESSOR_POLL_INTERVAL_MS", "value": "5000" }
```

Comment:
This is in preparation for #285; included it here since it's part of the environment variable documentation